core/src/main/java/org/apache/struts2/interceptor/csp/DefaultCspSettings.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.struts2.interceptor.csp;
+
+import com.opensymphony.xwork2.ActionContext;
+
+import javax.servlet.http.HttpServletResponse;
+import java.security.SecureRandom;
+import java.util.Base64;
+import java.util.Map;
+
+
+public class DefaultCspSettings implements CspSettings {
+
+    private static final int NONCE_LENGTH = 18;
+    private final SecureRandom sRand  = new SecureRandom();
+    private String reportUri = "csp-reports";
+    private boolean enforcingMode = false;
+    //TODO add string constants for csp header to avoid doing string operations each time
+
+    public void addCspHeaders(HttpServletResponse response) {
+        createNonce();
+        if (this.enforcingMode){
+            response.setHeader(CSP_ENFORCE_HEADER, getPolicyString());
+        } else {
+            response.setHeader(CSP_REPORT_HEADER, getPolicyString());
+        }
```

Comment:
let's simplify this code
```suggestion
        response.setHeader(enforcingMode ? CSP_ENFORCE_HEADER : CSP_REPORT_HEADER, getPolicyString());
```